students/ABartles/Exercise2/FizzBuzz.py

```diff
@@ -0,0 +1,10 @@
+for i in range(1, 101):
+
+    if (i % 3 == 0 and i % 5 == 0):
+        print("FizzBuzz")
+    elif (i % 3 == 0):
+        print("Fizz")
+    elif (i % 5 == 0):
+        print("Buzz")
+    else:
+        print(i)
```

Comment:
Looks good - make sure you submit this on the course site as well to receive credit.